scripts/compare_ci_diff_binables.sh

```diff
@@ -9,10 +9,11 @@ fi
 
 # cleanup if needed
 
-git clean -dfx
+git clean -dfx -e _opam/
```

Comment:
_opam should never be in the git tree as it is, wouldn't adding to `.gitignore` fix this?

Reply:
No. `git clean -x` does not respect standard ignore rules.
https://git-scm.com/docs/git-clean#Documentation/git-clean.txt--x